app/models/comment.rb

```diff
@@ -1,12 +1,10 @@
 class Comment < ApplicationRecord
-  belongs_to :user
   belongs_to :post
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
 
-  after_save :update_comments_counter
+  after_create :update_comment_counter
```

Comment:
* As mentioned in the [requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md#:~:text=Add%20unit%20specs%20for%20all%20of%20your%20models%27%20methods%20and%20validations.) you are to Add unit specs for all of your models' methods and validations. Kindly add test for like and comments models